theme_kiddo/models/theme_kiddo.py

```diff
@@ -6,10 +6,10 @@ class ThemeKiddo(models.AbstractModel):
 
     def _theme_kiddo_post_copy(self, mod):
         self.disable_view('website.template_header_default')
-        self.enable_view('website.template_header_centered_logo')
+        self.enable_view('website.template_header_default_align_right')
```

Comment:
There is something wrong here.
`template_header_default_align_right` is an option of the default header template.
But as `template_header_default` is disabled the line before, the menu from the portal module is shown.

<img width="1219" alt="Screenshot 2021-09-23 at 16 44 10" src="https://user-images.githubusercontent.com/7629949/134529051-da03bea3-5420-4627-8032-5142e90facda.png">

Reply:
Yeah... the align_right stuff is not important but if you disable all header template... well there is no menu :grimacing: